test/powershell/Language/Parser/Parsing.Tests.ps1

```diff
@@ -309,4 +309,11 @@ Describe 'Unicode escape sequence parsing' -Tag "CI" {
     ShouldBeParseError '"`u219z"' InvalidUnicodeEscapeSequence 1
     ShouldBeParseError '"`u{219z"' UnicodeEscapeSequenceMissingTerminator 7 # error offset "`u{219>>z<<"
     ShouldBeParseError '"`u{}"' InvalidBracketedUnicodeEscapeSequence 1
+    ShouldBeParseError '"`u' InvalidUnicodeEscapeSequence,TerminatorExpectedAtEndOfString 1,0
+    ShouldBeParseError '"`u123' InvalidUnicodeEscapeSequence,TerminatorExpectedAtEndOfString 1,0
+    ShouldBeParseError '"`u219z' InvalidUnicodeEscapeSequence,TerminatorExpectedAtEndOfString 1,0
+    ShouldBeParseError '"`u{' UnicodeEscapeSequenceMissingTerminator,TerminatorExpectedAtEndOfString 4,0
+    ShouldBeParseError '"`u{1' UnicodeEscapeSequenceMissingTerminator,TerminatorExpectedAtEndOfString 5,0
+    ShouldBeParseError '"`u{123456' UnicodeEscapeSequenceMissingTerminator,TerminatorExpectedAtEndOfString 10,0
+    ShouldBeParseError '"`u{1234567' UnicodeEscapeSequenceMissingTerminator,TerminatorExpectedAtEndOfString 10,0
```

Comment:
@lzybkr Should we report the error posion at '{'?

@rkeithhill What about tests for absent '{' ?  '"`u1234567}'